app/src/modules/collections/routes/collection.vue

```diff
@@ -365,6 +366,10 @@ export default defineComponent({
 
 		const { batchEditAllowed, batchArchiveAllowed, batchDeleteAllowed, createAllowed } = usePermissions();
 
+		useShortcut('delete', () => {
+			if (selection.value.length > 0) confirmDelete.value = true;
+		});
```

Comment:
On macOS, it's more common to have this be CMD+Backspace 🤔

Reply:
Seems like Ctrl+D would be more appropriate on windows

Reply:
Haha, didn't even knew Ctrl+D is a thing. 😄

Reply:
Can we / should we do Windows / Mac specific keyboard shortcuts 🤔 

Ctrl+D wouldn't make any sense on a mac (as ctrl isn't used for anything like that)

Reply:
I don't think so and I thought that `delete` would be a standard keyboard shortcut across all major platforms.

Reply:
macOS would use backspace, not delete 🥲